frontend-react/src/styles/_reportstream.scss

```diff
@@ -795,3 +795,49 @@ pre {
         margin-top: 40px;
     }
 }
+
+.rs-data-dashboard {
```

Comment:
kudos for encapsulating the custom style tweaks within a rs-data-dashboard container